testutil/kubernetes_helper.go

```diff
@@ -108,6 +120,20 @@ func (h *KubernetesHelper) CreateDataPlaneNamespaceIfNotExists(ctx context.Conte
 	return h.createNamespaceIfNotExists(ctx, namespace, annotations, map[string]string{"linkerd.io/is-test-data-plane": "true"})
 }
 
+// WithDataPlaneNamespace is used to create a test namespace that is deleted before the function returns
+func (h *KubernetesHelper) WithDataPlaneNamespace(ctx context.Context, testName string, annotations map[string]string, t *testing.T, helper *TestHelper, test func(t *testing.T, ns string)) {
```

Comment:
Doesn't it make more sense to declare this as a method on `TestHelper`, so you don't have to pass `*TestHelper` as an arg, and you have access to `KubernetesHelper` as a field?

Reply:
Yes it does indeed